### DISCUSSION TITLE
Could not  generate performance tree data

### DISCUSSION THREAD

**Discussion by w7658610** [openucx/ucx#8306] [Q&A] [answered]
Hi,
when I run mpi with hpcx-v2.11-gcc-MLNX_OFED_LINUX-5-redhat7-cuda11-gdrcopy2-nccl2.11-x86_64.tbz and the latest ucx .
But could not  generate performance tree data。
Is there something wrong with my operation?
```
mpirun --allow-run-as-root -np 2 -mca pml ucx -x UCX_LOG_FILE=/root/lktest/log/osu_log  -x UCX_PROTO_INFO=y -x UCX_PROTO_INFO_DIR=/root/lktest/log -x UCX_TLS=all -x UCX_NET_DEVICES=all -x LD_LIBRARY_PATH /opt/hpc/software/benchmark/osu-micro-benchmarks-5.5/libexec/osu-micro-benchmarks/mpi/pt2pt/osu_bw
```
>[root@d13r2n01 log]# ucx_info -v
>\# Library version: 1.14.0
>\# Library path: /root/lktest/ucx-master/install/lib/libucs.so.0
>\# API headers version: 1.14.0
>\# Git branch '', revision 
>\# Configured with: --prefix=/root/lktest/ucx-master/install

**Comment by yosefe** [comment, marked as answer]
@w7658610 pls add `-x UCX_PROTO_ENABLE=y` to enable new protocols
see also https://openucx.readthedocs.io/en/master/faq.html#protocol-selection
FYI, the performance tree data is still not fully merged to master branch, it will be added by #8282
